hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -2400,6 +2405,7 @@ public FlushResult flush(boolean force) throws IOException {
    * <p>This method may block for some time, so it should not be called from a
    * time-sensitive thread.
    * @param forceFlushAllStores whether we want to flush all stores
+   * @param families stores of region to flush.
```

Comment:
If null is passed, does that mean all families?

Reply:
Yeah, looks like it.

Reply:
So all families in a Region if 'families' is null or if forceFlushAllStores is true?

Reply:
No, only if forceFlushAllStores is true we select all families.